website/content/docs/secrets/databases/snowflake.mdx

```diff
@@ -75,6 +75,34 @@ The Snowflake database secrets engine uses
     from the root user you were provided when making your Snowflake account. This allows you to rotate
     the root credentials and still be able to access your account.
 
+1.  Configure Vault with keypair authentication:
+
+    ```shell-session
+    $ vault write database/config/my-snowflake-database \
+        plugin_name=snowflake-database-plugin \
+        allowed_roles="my-role" \
+        connection_url="ecxxxx.west-us-1.azure.snowflakecomputing.com/db_name" \
+        username="vaultuser" \
+        private_key=@key.pem
+    ```
+
+A properly formatted data source name (DSN) needs to be provided during configuration of the
+database. This DSN is typically formatted with the following options:
+
+```shell-session
+<account>.snowflakecomputing.com/<db_name>
+```
+
+`account` is your Snowflake account identifier. You can find out more about this value by reading
+the `server` section of
+[this document](https://docs.snowflake.com/en/user-guide/odbc-parameters.html#connection-parameters).
+
+`db_name` is the name of a database in your Snowflake instance.
```

Comment:
````suggestion
You must provide properly formatted data source names (DSN) when you cofigure
the database in the following format:

```text
<account>.snowflakecomputing.com/<db_name>
```

- `account` - your Snowflake account identifier. Refer to the
  [`server` section](https://docs.snowflake.com/en/user-guide/odbc-parameters.html#connection-parameters)
  of the connection parameters for Snowflake ODBC configuration and connection
  parameters.

- `db_name` the name of a database in your Snowflake instance.
````
Style correction: write in active voice, avoid "this" as a pronoun